src/js/components/AppVersionComponent.jsx

```diff
@@ -38,6 +40,40 @@ var AppVersionComponent = React.createClass({
     currentVersion: React.PropTypes.bool
   },
 
+  getInitialState: function () {
+    return {stale: false};
```

Comment:
For booleans I'd try to be consistent in using `is` or `has` + `ing` forms. How about `isStale` ?